addons/mail/static/src/components/message/message.js

```diff
@@ -689,6 +703,7 @@ Object.assign(Message, {
         hasCheckbox: Boolean,
         hasMarkAsReadIcon: Boolean,
         hasReplyIcon: Boolean,
+        hasUnfollowIcon: Boolean,
```

Comment:
It would be better if this was not a prop, but using a direct condition in the template instead (probably with thread_viewer thread === inbox?)